doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,255 @@
+    animation_frame=0,
+    height=500,
+    width=500,
+    binary_string=True,
+    labels={'animation_frame': 'time point'},
+    title='Sample of in-vivo human cornea'
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Average over time
+# =================
+# First, we want to detect those dark spots where the data are lost. In
+# technical terms, we want to *segment* the dark spots (for
+# all frames in the sequence). Unlike the actual data (signal), the dark spots
+# do not move from one frame to the next; they are still. Therefore, we begin
+# by computing the time average of the image sequence. We shall use this
+# time-averaged image to segment the dark spots, the latter then standing out
+# with respect to the background (blurred signal).
+
+image_avg = np.mean(image_seq, axis=0)
+
+print(f'shape: {image_avg.shape}')
+
+fig, ax = plt.subplots(figsize=(6, 6))
+
+ax.imshow(image_avg, cmap='gray')
+ax.set_title('Time-averaged image')
+
+fig.tight_layout()
+
+#####################################################################
+# Use local thresholding
+# ======================
+# To segment the dark spots, we use thresholding. The images we are working
+# with have uneven illumination, which causes variations in the (absolute)
+# intensities of the foreground and the background, from one region to another
+# (distant) one. It is therefore more fitting to compute different threshold
+# values across the image, one for each region. This is called adaptive (or
+# local) thresholding, as opposed to the usual thresholding procedure which
+# employs a single (global) threshold for all pixels in the image.
+#
+# When calling the ``threshold_local`` function from the ``filters`` module,
+# we may change the default neighborhood size (``block_size``), i.e., the
+# typical size (number of pixels) over which illumination varies,
+# as well as the ``offset`` (shifting the neighborhood's weighted mean).
+# Let us try two different values for ``block_size``:
+
+thresh_1 = filters.threshold_local(image_avg, block_size=21, offset=15)
+thresh_2 = filters.threshold_local(image_avg, block_size=43, offset=15)
+
+mask_1 = image_avg < thresh_1
+mask_2 = image_avg < thresh_2
+
+#####################################################################
+# Let us define a convenience function to display two plots side by side, so
+# it is easier for us to compare them:
+
+def plot_comparison(plot1, plot2, title1, title2):
+    fig, (ax1, ax2) = plt.subplots(
+        ncols=2,
+        figsize=(12, 6),
+        sharex=True,
+        sharey=True
+    )
+    ax1.imshow(plot1, cmap='gray')
+    ax1.set_title(title1)
+    ax2.imshow(plot2, cmap='gray')
+    ax2.set_title(title2)
+
+plot_comparison(mask_1, mask_2, "block_size = 21", "block_size = 43")
+
+#####################################################################
+# The "dark spots" appear to be more distinct in the second mask, i.e., the
```

Comment:
"dark spots" may not be the best name because after thresholding they appear as bright spots. I see a benefit in keeping with the paradigm large values are lighter, so pixels which are `True` should be white.  Suggestions in preferred order:
- Use a different term. What about "dust spots", "dirt spots",  or "obstruction spots". 
- Use a reversed colormap (in this case `"gray_r"`) for plots after thresholding, so that the "dark spots" are still dark.
- Add a sentence explaining why they now appear white.

Reply:
The first suggestion about replacing "dark spots" with "dust spots" adds more better explains the representation of the output plot.
@mkcor @ana42742 your thoughts?

Reply:
Then let's settle on "dirt spots". You are of course right that in the context of mask there is only True and False. Though, in this case we do represent it as "white spots". that's why I'd like to avoid a term that might be unnecessarily confusing for anyone that's not as familiar with these conventions... or just tired. :smile:

Reply:
Just noticed https://github.com/scikit-image/scikit-image/pull/6853#discussion_r1227080830. I'll leave it up to you to update the occurrences. It is very easy to loose track with this kind of workflow. Also who is supposed to actually commit suggestions?

Reply:
> Shall I push the update?
> It is very easy to loose track with this kind of workflow. Also who is supposed to actually commit suggestions?

Right. I have taken care of it with ac9fdb505560c. When we have two :+1: and no :-1: on a suggestion, we can commit. I may have been more expedient with the latest commits, interpreting some comments as upvotes.

PS: I'm happy to include some rewording along what @ana42742 suggested at https://github.com/scikit-image/scikit-image/pull/6853#discussion_r1227080830; it will be integrated with the workflow update (making it linear).

Reply:
One was left over! 620d11689273b4b4360f6fa0d4e54bfa98c75b39